madgiraffestudios/src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import './Gallery.scss';
+import ImageGallery from 'react-image-gallery';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+
+
+
+
+const Gallery = (props) => {
+
+  const gallery = [
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/nai_shoot.jpg?alt=media&token=5feb0e35-321c-4948-8f69-0f99f4e87b25' ,
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/nai_shoot.jpg?alt=media&token=5feb0e35-321c-4948-8f69-0f99f4e87b25',
+      startIndex: 0,
+      description: 'Nothing is ever black, white or gray. Let your colors shine! '
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/couple.jpg?alt=media&token=26bb85a4-956c-465e-a17b-06fb91120650',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/couple.jpg?alt=media&token=26bb85a4-956c-465e-a17b-06fb91120650',
+      startIndex: 1,
+      description: 'Strong Black Love'
+      
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      startIndex: 2,
+      description: 'A melanin queen '
+      
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
```

Comment:
- [ ] Please remove the duplicate images here, I believe this was done for testing only

Reply:
Resolved